src/remove.js

```diff
@@ -19,7 +19,6 @@ export default function remove(path, globalOptions, options) {
   const {
     visitedKey,
     wrapperIfTemplate,
-    mode,
```

Comment:
Mode is no longer a global option because the value change depending on `contextMode` and the `key` being a `contexTypes` or not. see:

``` js
 mode: isKeyContextTypes ? contextMode : mode,
```